readme.html

```diff
@@ -1,29 +1,468 @@
 <!-- File: readme.html
 
-Copyright (c) 2022 Illumio.
+    Copyright (c) 2022 Illumio.
 
-This unpublished material is proprietary to Illumio.
-All rights reserved. The methods and
-techniques described herein are considered trade secrets
-and/or confidential. Reproduction or distribution, in whole
-or in part, is forbidden except by express written permission
-of Illumio.
+    This unpublished material is proprietary to Illumio.
+    All rights reserved. The methods and
+    techniques described herein are considered trade secrets
+    and/or confidential. Reproduction or distribution, in whole
+    or in part, is forbidden except by express written permission
+    of Illumio.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
 
-  http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software distributed under
-the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
-either express or implied. See the License for the specific language governing permissions
-and limitations under the License. -->
+        http://www.apache.org/licenses/LICENSE-2.0
 
+    Unless required by applicable law or agreed to in writing, software distributed under
+    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied. See the License for the specific language governing permissions
+    and limitations under the License.
+-->
 <html>
 
-<head></head>
-
-<body>Replace this text in the app's <b>readme.html</b> to contain more detailed information</body>
+<body>
+    <p>
+        It is recommended to read the documentation for the app to understand the functioning of the actions and the
+        asset configuration or the action parameters associated with it. For further details, refer to <a
+            href='https://docs.illumio.com/core/21.5/API-Reference/index.html#Illumio-Core' target="_blank">Illumio
+            Docs</a>.
+    </p>
+    <h2>SDK Details</h2>
+    <p>
+        The app uses SDK for communicating with the Illumio server. Below is the SDK version used
+        by the Splunk SOAR Connector.
+    <table>
+        <tr>
+            <th>SDK NAME</th>
+            <th>VERSION</th>
+        </tr>
+        <tr>
+            <td>illumio</td>
+            <td>1.0.3</td>
+        </tr>
+    </table>
+    </p>
+    <p>
+    <h2>Steps to Configure the Illumio Phantom app's asset</h2>
+    <p>
+        Follow these steps to create an asset for the app in Phantom Platform:
+    <ul>
+        <li>Log in to the Illumio platform.</li>
```

Comment:
`Log in to the Illumio Policy Compute Engine`

Reply:
Done